signatures/golang/anti_debugging_ptraceme.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/aquasecurity/tracee/signatures/helpers"
+	"github.com/aquasecurity/tracee/types/detect"
+	"github.com/aquasecurity/tracee/types/protocol"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+type AntiDebuggingPtraceme struct {
+	cb            detect.SignatureHandler
+	ptraceTraceMe string
+}
+
+func (sig *AntiDebuggingPtraceme) Init(cb detect.SignatureHandler) error {
+	sig.cb = cb
+	sig.ptraceTraceMe = "PTRACE_TRACEME"
+	return nil
+}
+
+func (sig *AntiDebuggingPtraceme) GetMetadata() (detect.SignatureMetadata, error) {
+	return detect.SignatureMetadata{
+		ID:          "TRC-102",
```

Comment:
I notice that this changes the TRC numbers, is this on purpose? Should we update this to match what we have now? It might not matter

Reply:
He mentioned on teams, let me pick the chat...